src/tests/Button.test.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import renderer from 'react-test-renderer';
+import { render, cleanup } from '@testing-library/react';
+import Button from '../components/Button';
+import '@testing-library/jest-dom';
```

Comment:
You are not allowed to use this dependency, please remove it from all your tests and migrate them to use the @testing-library/react dependency.